### DISCUSSION TITLE
Prompt version comments

### DISCUSSION THREAD

**Discussion by mukundt** [langfuse/langfuse#1106] [Ideas]
### Describe the feature or potential improvement

Would be super helpful to have a comment box for each prompt version, to explain why something was changed.

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
That's a good one, thank you Mukund! will ping you here once we've added it.

**Comment by marliessophie** [comment]
We've released a first version of comments across objects in the Langfuse UI. Please see our [changelog](https://langfuse.com/changelog/2024-08-20-comments) for details. As always - we'd love to hear feedback on the feature!

Additionally, we believe that 'commit messages' for each prompt version are another avenue we'd like to additionally explore in the future. Excited to hear thoughts/input on this as well

**Comment by marliessophie** [comment]
We've released an initial version of commit messages for prompts, please see [changelog](https://langfuse.com/changelog/2025-01-28-prompt-commit-messages) post for details. Hope this helps!
